### DISCUSSION TITLE
Security Concerns with port forwarding Filebrowser

### DISCUSSION THREAD

**Discussion by wattsjake** [filebrowser/filebrowser#2922] [Q&A]
I'm running the latest version of Filebrowser on Proxmox as a docker container. I've had it running for about 2 months on my local network and I'd like to port forward it so I can use it outside of my network. I'd like to know the potential security issues that I will present to myself if I do port forward. I will say that I have set up a python/bash script that runs everyday and emails me the docker logs via 'journald'. This does allow me to view the ip addresses of devices that have tried to login. But this alone won't help me sleep at night.

Are there ways to get past the login screen? Can an attacker somehow gain access to the machine Filebrowser is running on (maybe gain root access)?

**Comment by Nerahikada** [comment]
I have no idea about the unknown vulnerabilities that exist in the filebrowser, but you can set it up so that only you can access it without port forwarding with Tailscale or Cloudflare Zero Trust.
